examples/Lorenz/Lorenz_example.jl

```diff
@@ -297,36 +302,34 @@ println(norm_factor)
 N_iter = 5;
 input_output_pairs = Utilities.get_training_points(ekiobj, N_iter)
 normalized = true
-
-# Default kernel
-gpobj = GaussianProcessEmulator.GaussianProcess(input_output_pairs, gppackage; 
-			 obs_noise_cov=Γy, normalized=normalized,
-                         noise_learn=false, 
-			 truncate_svd=truncate_svd, standardize=standardize,
-			 prediction_type=pred_type,
-			 norm_factor=norm_factor)
+emulator = Emulator(
+    gauss_proc,
+    input_output_pairs;
+    obs_noise_cov=Γy,
+    normalize_inputs=normalized,
+    standardize_outputs=standardize,
+    standardize_outputs_factors=norm_factor,
+    truncate_svd=truncate_svd
+)
+optimize_hyperparameters!(emulator)
 
 # Check how well the Gaussian Process regression predicts on the
 # true parameters
 #if truncate_svd==1.0
     if log_normal==false
-        y_mean, y_var = GaussianProcessEmulator.predict(gpobj, reshape(params_true, :, 1), 
-                                       transform_to_real=true)
+        y_mean, y_var = Emulators.predict(emulator, reshape(params_true, :, 1), 
+                                         transform_to_real=true)
     else
-    	y_mean, y_var = GaussianProcessEmulator.predict(gpobj, reshape(log.(params_true), :, 1), 
-                                       transform_to_real=true)
+    	y_mean, y_var = Emulators.predict(emulator, reshape(log.(params_true), :, 1), 
+                                         transform_to_real=true)
     end
     
     println("GP prediction on true parameters: ")
     println(vec(y_mean))
     println(" GP variance")
     println(diag(y_var[1],0))
     println("true data: ")
-    if standardize
-        println(truth.mean ./ norm_factor)
-    else
-        println(truth.mean)
-    end
+    println(truth.mean) # same, regardless of norm_factor
```

Comment:
We shouldn't have needed to rescale `truth.mean`, since that's what the emulator is emulating, and in the new code `y_mean` is correctly scaled by `reverse_standardize`. Verified that `vec(y_mean) ≈ truth.mean` in the new code, which means this was being done incorrectly (but consistently) on the existing code in `/master`.